ember-bootstrap/ember-cli-build.js

```diff
@@ -0,0 +1,14 @@
+'use strict';
+
+const EmberApp = require('ember-cli/lib/broccoli/ember-app');
+const { maybeEmbroider } = require('@embroider/test-setup');
+
+module.exports = function (defaults) {
```

Comment:
It seems you deliberately added this here in one of the last commits, but I wonder why? I don't think this is ever going to be used, as this file is purely app-related, and the app-part is now extracted into its own package.

Reply:
I added that while debugging the build issues, but forgot to remove it.
I've tested again with and without the file, just to be sure, and it does not make a difference, so I've removed it again.